llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp

```diff
@@ -1510,7 +1545,10 @@ static bool canMutatePriorConfig(const MachineInstr &PrevMI,
 
     auto &AVL = MI.getOperand(1);
     auto &PrevAVL = PrevMI.getOperand(1);
-    assert(MRI.isSSA());
+    assert(!AVL.isReg() || !AVL.getReg().isVirtual() ||
+           MRI.hasOneDef(AVL.getReg()));
```

Comment:
Is AVL and PrevAVL always OneDef?

Reply:
Apparently so, but I will double check. It could just be that we don't have tests for AVLs coming from a phi?

Reply:
I was able to create an .mir test case that fails this assertion at least. I think it should be ok if we just return false if there's more than one def?